public/css/modals/application-review-modal.css

```diff
@@ -55,6 +60,44 @@
 }
 
 i {
+	color: #50b5dd;
 	padding-left: 20px;
 	padding-right: 5px;
 }
+
+@media (max-width: 750px) {
```

Comment:
Bootstrap's grid system uses 768px as its mobile breakpoint; you should use the same for consistency, here and at any other media queries that are trying to target mobile devices (including in `admin-application-review.css`). You can see the problem this creates in the application bars that you click on to open the modal: if you shrink your screen to in-between 768px and 750px, the applicant's school and application status get shunted onto new lines